build.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+  id "org.owasp.dependencycheck" version "1.4.5"
+  id "com.github.johnrengelman.shadow" version "1.2.4"
```

Comment:
I got it but does it set the classpath as well? Also as i said before we need to explicitly set the repository as maven repo.